# 2017-02-06: harry.lin <harry.lin@deltaww.com>
# Change DDR config to support 1GB memory.
--- barebox-2011.06.0/arch/arm/mach-comcerto/include/mach/ddr.h.org	2017-01-19 15:48:33.778385150 +0800
+++ barebox-2011.06.0/arch/arm/mach-comcerto/include/mach/ddr.h	2017-01-19 15:48:58.395251880 +0800
@@ -624,11 +624,11 @@
 #define DDRC_CTL_DDR3_011_VAL_CFG2  0x00020010LL
 #define DDRC_CTL_DDR3_012_VAL_CFG2  0x00011B03LL
 #define DDRC_CTL_DDR3_013_VAL_CFG2  0x00000009LL
-#define DDRC_CTL_DDR3_014_VAL_CFG2  0x00560100LL
+#define DDRC_CTL_DDR3_014_VAL_CFG2  0x00A00100LL  //0x00560100LL
 #define DDRC_CTL_DDR3_015_VAL_CFG2  0x00001034LL
 #define DDRC_CTL_DDR3_016_VAL_CFG2  0x000D0004LL
 #define DDRC_CTL_DDR3_017_VAL_CFG2  0x00060002LL
-#define DDRC_CTL_DDR3_018_VAL_CFG2  0x005B0200LL
+#define DDRC_CTL_DDR3_018_VAL_CFG2  0x00A60200LL  //0x005B0200LL
 #define DDRC_CTL_DDR3_019_VAL_CFG2  0x00000000LL
 #define DDRC_CTL_DDR3_020_VAL_CFG2  0x00060600LL
 #define DDRC_CTL_DDR3_021_VAL_CFG2  0x00000000LL
@@ -638,9 +638,9 @@
 #define DDRC_CTL_DDR3_025_VAL_CFG2  0x00000000LL
 #define DDRC_CTL_DDR3_026_VAL_CFG2  0x00000000LL
 #define DDRC_CTL_DDR3_027_VAL_CFG2  0x00083000LL
-#define DDRC_CTL_DDR3_028_VAL_CFG2  0x00080006LL
+#define DDRC_CTL_DDR3_028_VAL_CFG2  0x00080042LL  //0x00080006LL
 #define DDRC_CTL_DDR3_029_VAL_CFG2  0x00000000LL
-#define DDRC_CTL_DDR3_030_VAL_CFG2  0x00060830LL
+#define DDRC_CTL_DDR3_030_VAL_CFG2  0x00420830LL  //0x00060830LL
 #define DDRC_CTL_DDR3_031_VAL_CFG2  0x00000008LL
 #define DDRC_CTL_DDR3_032_VAL_CFG2  0x00020000LL
 #define DDRC_CTL_DDR3_033_VAL_CFG2  0x00000000LL
@@ -656,7 +656,7 @@
 #define DDRC_CTL_DDR3_043_VAL_CFG2  0x01400200LL
 #define DDRC_CTL_DDR3_044_VAL_CFG2  0x02000040LL
 #define DDRC_CTL_DDR3_045_VAL_CFG2  0x01010080LL
-#define DDRC_CTL_DDR3_046_VAL_CFG2  0xFF0A0102LL
+#define DDRC_CTL_DDR3_046_VAL_CFG2  0xFF0A0101LL  //0xFF0A0102LL
 #define DDRC_CTL_DDR3_047_VAL_CFG2  0x010101FFLL
 #define DDRC_CTL_DDR3_048_VAL_CFG2  0x00010001LL
 #if defined(DDR_CFG_16BIT)
@@ -750,7 +750,7 @@
 #define DDR3_PHY_CTL_03_VAL_CFG2  0x09090909LL 
 #define DDR3_PHY_CTL_04_VAL_CFG2  0x00000009LL
 #define DDR3_PHY_CTL_05_VAL_CFG2  0x00000000LL
-#define DDR3_PHY_CTL_06_VAL_CFG2  0x04300623LL
+#define DDR3_PHY_CTL_06_VAL_CFG2  0x04300613LL  //0x04300623LL
 #define DDR3_PHY_CTL_07_VAL_CFG2  0x00000000LL
 
 
